org.rcpml.forms/src/org/rcpml/forms/internal/tags/ComboTagFactory.java

```diff
@@ -22,6 +22,15 @@ public ComboBridge(Node node, IController container) {
 
 		protected Combo constructCombo(Composite parent, int style) {
 			Combo combo = new Combo(parent, style);
+			String label = getAttribute("label");
+			combo.setToolTipText(label);
+			
+			String items = getAttribute("items");
+			if (items != null) {
+				for (String item : items.split(",")) {
+					combo.add(item);
+				}
+			}
```

Comment:
Is this code needed? Combo options are described using child <state> elements

Reply:
We have already sent examples using "items" attribute instead of "state" children elements to Renesas. Anyway it works only for combos without data binding, so I removed it